infrastructure/api-mgmt.tf

```diff
@@ -0,0 +1,48 @@
+locals {
+  # certificate thumbprints used by API tests
+  allowed_test_certificate_thumbprints = [
+    "${var.api_test_valid_certificate_thumbprint}",
+    "${var.api_test_expired_certificate_thumbprint}",
+    "${var.api_test_not_yet_valid_certificate_thumbprint}",
+  ]
+
+  # list of the thumbprints of the SSL certificates that should be accepted by the API (gateway)
+  allowed_certificate_thumbprints = "${concat(
+    local.allowed_test_certificate_thumbprints,
+    var.allowed_client_certificate_thumbprints
+  )}"
+
+  formatted_thumbprints = "${formatlist("&quot;%s&quot;", local.allowed_certificate_thumbprints)}"
+  quoted_thumbprints    = "${join(",", local.formatted_thumbprints)}"
+}
+
+module "api_mgmt_product" {
+  source        = "git@github.com:hmcts/cnp-module-api-mgmt-product.git"
+  api_mgmt_name = "core-api-mgmt-${var.env}"
+  api_mgmt_rg   = "core-infra-${var.env}"
+  name          = "${var.component}"
+}
+
+module "api_mgmt" {
+  source        = "git@github.com:hmcts/cnp-module-api-mgmt-api.git"
+  name          = "${var.component}-api"
+  api_mgmt_name = "core-api-mgmt-${var.env}"
+  api_mgmt_rg   = "core-infra-${var.env}"
+  revision      = "1" //???
```

Comment:
they're all 1 in prod, you can provide different API versions through this I think